README.md

```diff
@@ -33,6 +33,12 @@ If you have cutecord style notifications enabled, this is how we handle them.
 
 ---
 ## Changelog
+
+### v3.3.1
+Clearified what "Lurking guilds" Actually means, (No, it doesn't mean muted guilds!)
+Added option to still get notified by cuties even if they said something in a muted guild
+Added option to still get notified by cuties even if you can already see them (In the current chat)
+Fixed case insensitive not working with capatalized cute words
```

Comment:
Match the style of the rest of the file please - no lines longer than 80 characters. In addition, displaying a newline in markdown requires two newlines. If you look at the preview on your branch, your whole changelog is one line.

Reply:
Noted, I was editing the MD through a terminal at the time so didn't see it was all on one line. Re formatted it to use unordered lists and under 80char per line